### DISCUSSION TITLE
XBB成最強病毒株 醫：打非次世代第四劑、確診過都沒用

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#997] [General]
剛剛看到的，疫情一直在進化，根本就是沒完沒了的在抗議，就像是一個遊戲<瘟疫公司>根本無止盡，疫情到底要到什麼時候才會真正的結束，我已經不想再過著一直戴口罩的生活了ㄚㄚㄚㄚ
然後關於疫苗的部分我自己是覺得兩季就夠了，我家裡的人打了超過兩劑的人都確診了，我個人覺得是運氣好壞的問題啦，所以那種新聞說打幾劑都沒有用，真的是聽聽就好。
https://www.msn.com/zh-tw/news/living/xbb%E6%88%90%E6%9C%80%E5%BC%B7%E7%97%85%E6%AF%92%E6%A0%AA-%E9%86%AB-%E6%89%93%E9%9D%9E%E6%AC%A1%E4%B8%96%E4%BB%A3%E7%AC%AC%E5%9B%9B%E5%8A%91-%E7%A2%BA%E8%A8%BA%E9%81%8E%E9%83%BD%E6%B2%92%E7%94%A8/ar-AA15Ze7u?ocid=msedgntphdr&cvid=3d23830f12544a66bc02c8dab7dc09d3

**Comment by Floriaand** [comment]
其實已經戴口罩習慣了，得到了也還好就是嚴重一點的感冒，只是會擔心在考試前後得到，補考真的太麻煩了。

**Comment by swerfg03joe** [comment]
口罩戴了將近四年，期間完全沒有感冒過，我覺得病毒這種東西也不會憑空出現，最重要的是該如何保護自己，疫苗研發速度一定趕不上病毒變異速度，再說病毒都流行這麼久了致死率應該下降不少

**Comment by harrylee1971** [comment]
管他的，我覺得該帶口罩就帶，像是室外空間的話就不用戴口罩了

**Comment by Jzd314** [comment]
口罩戴好，保持健康的衛生習慣才是最重要的吧
